openbb_terminal/terminal_helper.py

```diff
@@ -355,6 +356,7 @@ def reset(queue: Optional[List[str]] = None):
     logger.info("resetting")
     plt.close("all")
     plots_backend().close(reset=True)
+    load_env_files()
```

Comment:
Conflicts with https://github.com/OpenBB-finance/OpenBBTerminal/pull/4633, but can be easily fixed on whichever gets merged last.